vidarrtest-regression.json.in

```diff
@@ -10,7 +10,7 @@
             {
                 "metrics_calculate": "@CHECKOUT@/tests/calculate.sh",
                 "metrics_compare": "@CHECKOUT@/tests/compare.sh",
-                "output_metrics": "/.mounts/labs/gsi/testdata/emSeqQc/expected_output/single.txt",
```

Comment:
These should not be changing as they have some useful information about the test (and do not contain any sensitive knowledge).